integration-tests/src/test/java/org/apache/fineract/integrationtests/DelinquencyBucketsIntegrationTest.java

```diff
@@ -781,7 +781,7 @@ public void testLoanClassificationStepAsPartOfCOB() {
 
         // Move the Business date to get older the loan and to have an overdue loan
         LocalDate lastLoanCOBBusinessDate = businessDate;
-        businessDate = businessDate.plusDays(3);
+        businessDate = businessDate.plusDays(4);
```

Comment:
I dont like this.

If it was not failing prior, why it started to fail when the catch up was introduced.

If it was wrong, why it was not failing and why started now?